components/frontend/guestRoomDetailsPage/HeroBanner.tsx

```diff
@@ -98,57 +71,41 @@ const HeroBanner = () => {
               }}
             />
             <Typography
-              variant="body1"
-              className={mulish.className}
-              style={checkBoxTypographyStyle}
+              variant="h4"
+              color= "#9A9AB0"
+              marginTop= "39px"
             >
               {roomDetailsHeroSection.checkIcon3}
             </Typography>
           </Box>
           <Typography
-            variant="body2"
-            className={mulish.className}
-            style={{
-              color: "#515151",
-              fontSize: "14px",
-              lineHeight: "130%",
-              fontStyle: "normal",
-              letterSpacing: "0.112px",
-              fontWeight: "600",
-              marginTop: "36px",
-            }}
+            variant="h5"
+            color="#515151"
+            fontWeight="700"
+            marginTop="36px"
           >
             {roomDetailsHeroSection.customText1}
           </Typography>
           <Box sx={{ display: "flex" }}>
             <Typography
-              variant="body2"
-              className={lora.className}
-              style={{
-                color: "#27343F",
-                fontSize: "39px",
-                lineHeight: "120%",
-                fontStyle: "normal",
-                letterSpacing: "0.195px",
-                fontWeight: "700",
-                marginTop: "4px",
-              }}
+              fontSize="39px"
+              fontFamily="Mulish"
+              color="#27343F"
+              lineHeight="120%"
+              fontStyle="normal"
+              letterSpacing="0.195px"
+              fontWeight= "700"
+              marginTop= "4px"
             >
               {roomDetailsHeroSection.price}
             </Typography>
             <Typography
-              variant="body2"
-              className={mulish.className}
-              style={{
-                color: "#9A9AB0",
-                fontSize: "12px",
-                lineHeight: "120%",
-                fontStyle: "normal",
-                letterSpacing: "0.096px",
-                fontWeight: "400",
-                marginTop: "22px",
-                marginLeft: "4px",
-              }}
+              variant="h6"
+              color= "#9A9AB0"
+              letterSpacing= "0.096px"
```

Comment:
Instead of adding different values for properties such as letterSpacing: "0.096px", try to use a common one via the theme.